src/modules/billing/services/billing-volumes-service.js

```diff
@@ -0,0 +1,82 @@
+const billingVolumesRepo = require('../../../lib/connectors/repos/billing-volumes');
+const mappers = require('../mappers');
+const twoPartTariffMatching = require('./two-part-tariff-service');
+const { isEmpty, flatMap } = require('lodash');
+
+const chargeElementSeasonMatches = (chargeElement, isSummer) => {
+  const chargeElementIsSummer = chargeElement.season === 'summer';
+  return chargeElementIsSummer === isSummer;
+};
+
+const getChargeElementIds = (chargeVersion, isSummer) => {
+  return chargeVersion.chargeElements.filter(chargeElement => {
+    if (chargeElementSeasonMatches(chargeElement, isSummer)) {
+      return chargeElement.chargeElementId;
+    };
+  });
+};
```

Comment:
I struggled a bit to understand this, and I don't think it is returning an array of IDs.
Maybe this might be simpler:
```
const isSummerChargeElement = chargeElement => chargeElement.season === 'summer';

const getChargeElementsForSeason = (chargeVersion, isSummer) => {
  return chargeVersion.chargeElements
   .filter(chargeElement => isSummerChargeElement(chargeElement) === isSummer);
}
```

Reply:
This was a map method, but I clearly changed it and didn't update the return value